bin/tictactoe

```diff
@@ -1,3 +1,27 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+
```

Comment:
I would abstract lines 5 - 27 into a CLI class with methods in the `lib` folder and on line 5 just call 
```ruby 
CLI.new.start_game
```